code/assurance/src/main/java/com/adobe/marketing/mobile/assurance/AssuranceSession.java

```diff
@@ -165,8 +165,25 @@ void connect(final String pin) {
 		assuranceSessionPresentationManager.onSessionConnecting();
 
 		final String envString = AssuranceUtil.getURLFormatForEnvironment(assuranceEnvironment);
+		String orgId = assuranceStateManager.getOrgId(true);
+
+		if (StringUtils.isNullOrEmpty(orgId)) {
+			// Configuration may not have set the state with org id yet.
```

Comment:
Can you reword the comment to some thing like "If configuration does not have org-id, try extracting it from stored connection url"